.cookiecutter/cookiecutter.json

```diff
@@ -0,0 +1,31 @@
+{
+  "template": "https://github.com/hypothesis/cookiecutters",
+  "checkout": null,
+  "directory": "pyapp",
+  "ignore": ["conf/supervisord*.conf", "h_periodic/__main__.py", "h_periodic/app.py", "h_periodic/core.py", "tests/unit/h_periodic/core_test.py"],
```

Comment:
The `conf/supervisord*.conf` templates in the cookiecutter are good enough (and necessary) for generating new projects but they're not really good enough to be used in existing projects yet. Need to improve those templates in the cookiecutter. For now, h-periodic can ignore those two template files and just supply its own Supervisor config files manually, and still benefit from the rest of the cookiecutter.

The other ignored files here are starter files that're necessary for newly-created apps to have working hello-world `make dev` and `make docker-run` commands but that h-periodic doesn't want.